voice-interface/src/main/java/com/derpgroup/derpwizard/voice/model/SsmlDocumentBuilder.java

```diff
@@ -82,6 +82,7 @@ public String toString() {
   private Set<String> ignoreTags;
   private List<List<StringBuilder>> paragraphs;
   private int index = 0;
+  private boolean conversationEnd = true;
```

Comment:
I happen to agree.  We just don't really have a better place for it, other than maybe stashing it in the metadata.